diagnostic.md

```diff
@@ -53,7 +65,14 @@ How would we reference each of the following categories of elements?
 -   Only those elements belonging to both the `important` and `red` _classes_
 -   All `p` elements whose parent elements are `div`s
 
+
 <!-- your answer starts here -->
+```
+$('.big');
+$('#contentPane');
+$('div').children;
+$('.important .red');
+```
```

Comment:
Want to not be in jQuery land for this. Your selectors are on the right track, though.

```CSS
.big
#contentPane
.important.red
div > p
```